<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PropagationCalculator</class>
 <widget class="QMainWindow" name="PropagationCalculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1351</width>
    <height>871</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Propagation Calculator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="pushButton">
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>Load File</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="auto_analysis">
      <property name="text">
       <string>Auto Analysis</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4" rowspan="9">
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>1041</width>
        <height>671</height>
       </size>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::East</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <property name="movable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="tabWidgetPage1">
       <attribute name="title">
        <string/>
       </attribute>
      </widget>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QProgressBar" name="progressBar">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1351</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_Data"/>
    <addaction name="actionSave_Data"/>
    <addaction name="actionExport_Graphic"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionAlter_Markers"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionShow_Axes"/>
    <addaction name="actionShow_Data"/>
   </widget>
   <widget class="QMenu" name="menuRun">
    <property name="title">
     <string>Variables</string>
    </property>
    <addaction name="actionAbsolute_Value_2"/>
    <addaction name="actionFilter"/>
    <addaction name="actionFilter_Config"/>
   </widget>
   <widget class="QMenu" name="menuAutomatic_Analysis">
    <property name="title">
     <string>Automatic Analysis</string>
    </property>
    <addaction name="actionAnalyse_waveform_currently_loaded"/>
    <addaction name="actionAnalyse_multiple_additional_files"/>
    <addaction name="actionSet_threshold"/>
    <addaction name="actionSet_width"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuRun"/>
   <addaction name="menuAutomatic_Analysis"/>
  </widget>
  <action name="actionAxon_binary_file">
   <property name="text">
    <string>axon binary file</string>
   </property>
  </action>
  <action name="actionOpen_Data">
   <property name="text">
    <string>Open Data</string>
   </property>
  </action>
  <action name="actionSave_Data">
   <property name="text">
    <string>Save Data</string>
   </property>
  </action>
  <action name="actionExport_Graphic">
   <property name="text">
    <string>Export Graphic</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionShow_Axes">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Axes</string>
   </property>
  </action>
  <action name="actionShow_Data">
   <property name="text">
    <string>Show Data</string>
   </property>
  </action>
  <action name="actionAbsolute_Value">
   <property name="text">
    <string>Absolute Value</string>
   </property>
  </action>
  <action name="actionAlter_Markers">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Alter Markers</string>
   </property>
  </action>
  <action name="actionAbsolute_Value_2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Absolute Value</string>
   </property>
  </action>
  <action name="actionFilter">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Filter</string>
   </property>
  </action>
  <action name="actionFilter_Config">
   <property name="text">
    <string>Filter Config</string>
   </property>
  </action>
  <action name="actionAnalyse_waveform_currently_loaded">
   <property name="text">
    <string>Analyse file currently loaded</string>
   </property>
  </action>
  <action name="actionAnalyse_multiple_additional_files">
   <property name="text">
    <string>Analyse multiple additional files</string>
   </property>
  </action>
  <action name="actionSet_threshold">
   <property name="text">
    <string>Set threshold</string>
   </property>
  </action>
  <action name="actionSet_width">
   <property name="text">
    <string>Set window</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
